internal/chart/process.go

```diff
@@ -125,12 +132,75 @@ func withVolumeMounts(vm []v1.VolumeMount) processOption {
 	}
 }
 
+// withLabels returns a function that populates Kind labels which specify a matching/unspecified deploymentVersion, a matching/unspecified
+// processName, and apiVersion v1.
+func withLabels(labels []ketchv1.MetadataItem, deploymentVersion ketchv1.DeploymentVersion) processOption {
+	return func(p *process) error {
+		for _, label := range labels {
+			if label.DeploymentVersion > 0 && int(deploymentVersion) != label.DeploymentVersion ||
+				label.ProcessName != "" && label.ProcessName != p.Name ||
+				label.Target.APIVersion != "v1" {
+				continue
+			}
+			if err := label.Validate(); err != nil {
+				return err
+			}
+			for k, v := range label.Apply {
+				switch label.Target.Kind {
```

Comment:
Kind seems to just be string, so do you think it would be a good idea to add a default case to this switch statement?  Or maybe kind could make use of go's "enum" functionality to guarantee that only Deployment and Service can be used.

Reply:
That's a good point. I added an error on the default case. I _think_ the intent is to only permit labels/annotations on Services and Deployments. Users should be notified if they accidentally try something else.